src/components/common/dashboard/Navbar.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Link, Button, Flex, Heading, Icon, Image, VStack } from '@chakra-ui/react';
+import { FiUsers } from 'react-icons/fi';
+import { IoHomeOutline, IoPodiumOutline } from 'react-icons/io5';
+import { Link as RouterLink, NavLink } from 'react-router-dom';
+import Logo from '../../../assets/logoWhite.png';
+
+const NavItem = props => (
+  <Link as={NavLink} to={props.to} _activeLink={{ color: 'white' }} _hover={{ color: 'white' }} textDecoration="none">
+    <Heading size="h4" as="span">
+      <Flex>
+        <Icon as={props.icon} mr={2} />
+        {props.children}
+      </Flex>
+    </Heading>
+  </Link>
+);
+
+const Navbar = props => (
+  <Flex
+    as="nav"
+    bg="background.secondary"
+    direction="column"
+    h="100%"
+    px={12}
+    py={20}
+    justify="space-between"
+    overflowX="hidden"
+    textTransform="capitalize"
+    role="navigation"
+  >
+    <VStack color="gray.400" spacing={10} align="flex-start">
+      <Image src={Logo} alt="Raising The Roof Logo" w="100%" />
+      <NavItem to="/home" icon={IoHomeOutline}>
+        Home
+      </NavItem>
+      <NavItem to="/leaderboard" icon={IoPodiumOutline}>
+        Leaderboard
+      </NavItem>
+      <NavItem to="/team" icon={FiUsers}>
+        Team
+      </NavItem>
+      <Button color="black" w="100%">
+        Share Storefront
+      </Button>
+    </VStack>
+    <Link as={RouterLink} to="/logout">
```

Comment:
one more change and then we should be good to merge

can you change this to a button that dispatches the `logout` action? I think top-level routing logic will automatically route to the login page as a result of that action (because `user` is set to `null` in Redux)

then we can completely remove the `/logout` path and the `Logout` page too!